spec/inputs/syslog_spec.rb

```diff
@@ -1,5 +1,5 @@
 # encoding: utf-8
-require "logstash/devutils/rspec/spec_helper"
+require_relative "../spec_helper"
```

Comment:
generally speaking yes but specifically here I don't agree, for 3 reasons:
1- the same fix we applied to the 2-3 other plugins are straigth in the main spec file and in this context I think it's better to stay consistent, knowing we will also revisit this patch soon.
2- adding extra structure (1 extra file, plus extra require) is useless for such a small patch.
3- I would argue that this patch is better placed at the top of this spec as it makes it more obvious to understand what is happening. 

of course it would make sense to put it in a helper if there were multiple spec that require that path but it's not the case here and there is no plan to create new spec file in the near future and at that point that patch will have matured and changed.